trackstop.lua

```diff
@@ -52,6 +52,23 @@ local DIRECTION_MAP = {
 
 local DIRECTION_MAP_REVERSE = utils.invert(DIRECTION_MAP)
 
+local function swapElements(tbl, index1, index2)
+  tbl[index1], tbl[index2] = tbl[index2], tbl[index1]
+  return tbl
+end
+
+local function reset_guide_paths(conditions)
+  for _i, condition in ipairs(conditions) do
```

Comment:
for unused variables, the Lua convention is just to use an underscore with no other characters.

Reply:
Ah yeah that's it. Was wondering why it still showed greyed out in the editor but hadn't looked into it yet. ty!